packages/blade/src/components/Box/Box.tsx

```diff
@@ -0,0 +1,104 @@
+import BaseBox from './BaseBox';
+import type { BoxProps } from './BaseBox/types';
+
+/**
+ * This function is to filter out any unexpected props passed by the user
+ */
+const getOnlyBoxProps = (props: BoxProps): BoxProps => {
+  return {
```

Comment:
Can you also add comment blocks in between, would make it easier to parse all the props. Eg.

```jsx
// flex
...

// layout
...
```

Reply:
yay yes. done